### DISCUSSION TITLE
Gzip is enabled in developer tools but not shows up in light house.

### DISCUSSION THREAD

**Discussion by mobitsolutions-dev** [GoogleChrome/lighthouse#15713] [Q&A]
Hi,

Here is the website is am working on
Website: https://www.mayfairparking.co.uk

gzip is enabled and chrome shows in developer tools which is fine and as per my expectations but when i test on light house it gives me text compression error message see screenshots below.

Light House Error:
![image](https://github.com/GoogleChrome/lighthouse/assets/67701415/80058a1e-6566-4a23-a873-29e50bed6e4c)

Developer Tools Screenshot:
![image](https://github.com/GoogleChrome/lighthouse/assets/67701415/a2e24ee9-0ace-49d3-b989-04829a7e2a55)

any one can suggest why this is happening? and what is the possible fix?

any help would be appreciated.

**Comment by adamraine** [comment]
https://developer.chrome.com/docs/lighthouse/performance/uses-text-compression/

The client correctly populates the `Accept-Encoding` request header, but the server still did not compress the data because it did not populate the `Content-Encoding` response header.
